src/core/dbus-cgroup.c

```diff
@@ -1711,6 +1714,83 @@ int bus_cgroup_set_property(
                 return 1;
         }
 
+        if (streq(name, "AllowBindPorts")) {
+                Set **ports = &(c->allow_bind_ports);
+                size_t n = 0;
+
+                r = sd_bus_message_enter_container(message, 'a', "s");
+                if (r < 0)
+                        return r;
+
+                for (;;) {
+                        const char *s;
+                        uint16_t port;
+
+                        r = sd_bus_message_read(message, "s", &s);
+                        if (r < 0)
+                                return r;
+
+                        if (r == 0)
+                                break;
+
+                        if (streq(s, "none")) {
```

Comment:
Parenthesis not needed here

Reply:
@anitazha 

`if` is followed by another `if` so parenthesis are needed
```
                        if (streq(s, "none")) {
                                if (!UNIT_WRITE_FLAGS_NOOP(flags))
                                        set_clear(*ports);
                        } else {
```